test/verify/check-openshift

```diff
@@ -652,7 +652,7 @@ class TestRegistry(MachineCase):
             b.click(".btn-primary")
             b.wait_not_present("modal-dialog")
             b.wait_present("table.listing-ct")
-            wait(lambda: username not in o.execute("oc get rolebinding -n testprojectuserproj"))
+            wait(lambda: username not in o.execute("oc get rolebinding -n testprojectuserproj"), delay=5)
```

Comment:
Don't forget that our test runners can be under significant load.
We don't really care about testing how fast OpenShift is - we want this test to fail if the Cockpit UI to it is broken.
Increasing the delay is good to reduce the polling frequency. We could lower the number of tries. Realistically this test will either fail completely if something is broken, otherwise we're not going to have much slower tests due to this change (rather more saved time due to not having to redo the entire suite).